examples/documents/paged_results.py

```diff
@@ -0,0 +1,15 @@
+# [START import]
+from nitric.api import Documents
+# [END import]
+async def documents_paged_results():
+# [START snippet]
+    docs = Documents()
+
+    query = docs.collection("Customers").query().where("active", "==", True).limit(100)
+
+    # Fetch first page
+    results = query.fetch()
+
+    # Fetch next page
+    results = query.page_from(results.paging_token).fetch()
```

Comment:
missing `await` on these two calls to fetch. Also, I'd suggest wrapping the 'fetch next page' in an if, e.g.

```python
if results.paging_token:
  results = await query.page_from(results.paging_token).fetch()
```

To help make it clear that the paging token is only present when more pages are left to fetch.